src/main/java/fi/aalto/cs/intellij/notifications/BetaVersionWarning.java

```diff
@@ -0,0 +1,32 @@
+package fi.aalto.cs.intellij.notifications;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import fi.aalto.cs.intellij.common.Version;
+import org.jetbrains.annotations.NotNull;
+
+public class BetaVersionWarning extends Notification {
+  private final Version version;
+
+  /**
+   * Constructs a notification that warns a user that they are using a pre-release version of the
+   * software.
+   * @param version Version of the software.
+   */
+  public BetaVersionWarning(@NotNull Version version) {
+    super("A+",
+        "A+ Courses plugin is under development",
+        "You are using version " + version + " of A+ Courses plugin, "
+            + "which is a pre-release version of the plugin and still under development. "
+            + "Some features of this plugin are still probably missing, "
+            + "and the plugin is not yet tested thoroughly. "
+            + "Use this plugin with caution and on your own risk!",
+        NotificationType.WARNING);
+    this.version = version;
+  }
+
+  @NotNull
+  public Version getVersion() {
+    return version;
+  }
+}
```

Comment:
does this have a use case beyond testing?

Reply:
Perhaps not. The point here was that because a `BetaVersionWarning` object (in its initial state) can be fully described by the version it was given in the constructor (that is, two `BetaVersionWarning` objects that has the same `version` field do not differ each other in their initial states) it is nice to provide an access to that field.